src/NLPModels.jl

```diff
@@ -172,6 +173,19 @@ function objgrad!(nlp, x, g)
   return f, g
 end
 
+"""`(rows,cols) = jac_structure(nlp)`
```

Comment:
I think the calling sequence should be on the second line, indented 4 spaces and with no backticks.

Reply:
Most docstrings in `NLPModels.jl` are this way. I'll open an issue to change them afterwards.

Reply:
Ok